Jenkinsfile-release-config

```diff
@@ -0,0 +1,44 @@
+pipeline{
+	agent any
+	
+	stages{
+		stage('Confirm configurations'){
+			steps{
+				script{
+          				def userInput = input(
+          				id: 'userInput', message: "Has the Config file credentials been updated for this release?",
+          				parameters: [
```

Comment:
It looks like this line (and many others) use a mix of tabs and spaces. As this is all new code, could we have the tabs/spaces made consistent within the file? Either all tabs, or all spaces, I'm not _too_ picky. But mixing tabs and spaces can cause weird & annoying formatting issues later.